README.md

```diff
@@ -0,0 +1,289 @@
+|                                    |                   | [SetHostname](https://www.onvif.org/ver10/device/wsdl/devicemgmt.wsdl#op.SetHostname)                                           | Object           |
+|                                    |                   | [GetDNS](https://www.onvif.org/ver10/device/wsdl/devicemgmt.wsdl#op.GetDNS)                                                     | Object           |
+|                                    |                   | [SetDNS](https://www.onvif.org/ver10/device/wsdl/devicemgmt.wsdl#op.SetDNS)                                                     | Object           |
+|                                    |                   | [**GetNetworkInterfaces**](https://www.onvif.org/ver10/device/wsdl/devicemgmt.wsdl#op.GetNetworkInterfaces)                     | Object           |
+|                                    |                   | [**SetNetworkInterfaces**](https://www.onvif.org/ver10/device/wsdl/devicemgmt.wsdl#op.SetNetworkInterfaces)                     | Object           |
+|                                    |                   | [GetNetworkProtocols](https://www.onvif.org/ver10/device/wsdl/devicemgmt.wsdl#op.GetNetworkProtocols)                           | Object           |
+|                                    |                   | [SetNetworkProtocols](https://www.onvif.org/ver10/device/wsdl/devicemgmt.wsdl#op.SetNetworkProtocols)                           | Object           |
+|                                    |                   | [**GetNetworkDefaultGateway**](https://www.onvif.org/ver10/device/wsdl/devicemgmt.wsdl#op.GetNetworkDefaultGateway)             | Object           |
+|                                    |                   | [**SetNetworkDefaultGateway**](https://www.onvif.org/ver10/device/wsdl/devicemgmt.wsdl#op.SetNetworkDefaultGateway)             | Object           |
+| System Function                    | Device            | [**GetDeviceInformation**](https://www.onvif.org/ver10/device/wsdl/devicemgmt.wsdl#op.GetDeviceInformation)                     | Object           |
+|                                    |                   | [GetSystemDateAndTime](https://www.onvif.org/ver10/device/wsdl/devicemgmt.wsdl#op.GetSystemDateAndTime)                         | Object           |
+|                                    |                   | [SetSystemDateAndTime](https://www.onvif.org/ver10/device/wsdl/devicemgmt.wsdl#op.SetSystemDateAndTime)                         | Object           |
+|                                    |                   | [SetSystemFactoryDefault](https://www.onvif.org/ver10/device/wsdl/devicemgmt.wsdl#op.SetSystemFactoryDefault)                   | Object           |
+|                                    |                   | [Reboot](https://www.onvif.org/ver10/device/wsdl/devicemgmt.wsdl#op.Reboot)                                                     | Object           |
```

Comment:
Link should be https://www.onvif.org/ver10/device/wsdl/devicemgmt.wsdl#op.SystemReboot

Reply:
Fixed the wrong link.